### PR TITLE
Editorial

### PR DESCRIPTION
**Pull request by tkopczuk**

### CODE REVIEW

File: What’s up in Vietnam.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+Following the definition on Wikipedia _it’s a technique for converting data between incompatible type systems using object-oriented programming languages._ To be more precise it’s converting relational data to the object-oriented domain model (OODM) and OODM to relational data. Conversions like that can be done in several ways, but the most common approach is to use a tool called ORM library or ORM framework.
 
 ## Why is it problematic?
- „Incompatible type systems” are mentioned already in the definition, and all incompatibilities are potential problems. My first question that popped into my head after getting into the subject was: „Why the hell we are using incompatible systems? Why is it the first (and often the only one known) solution to solve the problem with persisting data?”
+ „Incompatible type systems” are mentioned already in the definition, and all incompatibilities are potential problems. First question that popped into my head after getting into the subject was: „Why the hell are we using incompatible systems? Why is it the first (and often the only) solution to solve the problem with persisting data?”
```
Comment (dawiddominiak) [inline What’s up in Vietnam.md:9]: `the only` without `the only [one] known` changes context a little bit. The question is: is it bad ... hmmm.

File: What’s up in Vietnam.md [hunk 2]
```diff
@@ -45,27 +45,27 @@ kate = new User("Kate")
...
+### Table-per-concrete-class approach <@FIXME: elaborate more on this approach :)>
+This approach has similar disadvantages. Let’s try to retrieve ten most recently updated devices. We need to union all data in a really obscure way (creating many Null columns):
 ```
 SELECT Col1, Col2, Null as Col3 FROM light_management_devices,
 UNION Col1, Null as Col2, Col3 FROM heating_devices.
 ```
 … and then order by production date. It’s even more impractical and inefficient.
 
-### Table-per-hierarchy approach
+### Table-per-hierarchy approach <@TODO: I'd debate the first sentence>
```
Comment (dawiddominiak) [inline What’s up in Vietnam.md:68]: It's my experience ;) IDK if my experience is debatable.

File: What’s up in Vietnam.md [hunk 3]
```diff
@@ -107,19 +108,19 @@ _Ted Neward_
...
+1. Use a proper database to your need. Maybe it’s better to use a document storage/key-value store (like in the example with smart devices), a graph database (in many social services) or a completely different type of database.
 2. Use CQRS, save events as known truths and visualize data in reporting stores in proper databases: relational, context-dependent data in RDBS, objects in key-value stores, graphs in graph databases etc.
-3. Use half-way solution shying away from ORM frameworks, but still using RDBS. It’s recommended approach if we want to keep the clear separation between domain and persistence layer without additional maintenance of persistence model needed for ORM framework. In exchange, we can use the database with plain SQL more freely without the risk of side effects.
+3. Use a half-way solution shying away from ORM frameworks, but still using RDBS. It’s a recommended approach if we want to keep a clear separation between domain and persistence layer without additional maintenance A persistence model needed for ORM framework would bring. In exchange, we can use the database with plain SQL more freely without the risk of causing side effects.
```
Comment (dawiddominiak) [inline What’s up in Vietnam.md]: You sure about this capital A in the middle?

Comment (tkopczuk) [reply What’s up in Vietnam.md]: I'm sure it's wrong :d 'of a' is what  I was looking for I think.

File: What’s up in Vietnam.md (hunk 1 above)
Comment (tkopczuk) [reply What’s up in Vietnam.md:9]: Question is what is it that you wanted to say. 'The only one' suggests it is the only one, 'quite often the only one ' makes no sense I think, hence I added 'known' :) Probably just needs to be rephrased to pass carry the meaning you wanted it to carry.

**Pull request merged**